landing/public/css/home.css

```diff
@@ -2,151 +2,519 @@ html {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
-  overflow: hidden;
+  background-color: #fff;
+  overflow-x: hidden;
+  overflow-y:auto;
+  overscroll-behavior: none;
+  /* height: auto; */
+  /* overflow: hidden; */
+}
+
+section {
+  z-index: 1;
+}
+
+.primary-white-btn {
+  background: #fff;
+  box-shadow: var(--regular-box-shadow);
+  transition: background 200ms;
+
+}
+
+.primary-white-btn:hover {
+  background: var(--primary-main);
+  box-shadow: var(--regular-box-shadow);
+  transition: background 200ms;
+
+}
+
+.primary-white-btn-link {
+  font-weight: 700;
+  color: var(--primary-main);
+}
+
+.primary-white-btn-link:hover {
+  font-weight: 700;
+  color: #fff;
+}
+
+.heroBG {
+  position: absolute;
+  z-index: -4;
+  mix-blend-mode: multiply;
+  width: 100%;
+  height: auto;
+  max-height: 100vh;
+  opacity: 70%;
+}
+
+.heroCircle1 {
+  position: absolute;
+  top: 8px;
+  z-index: -5;
+  max-width: 80px;
+}
+
+.heroCircle2 {
+  position: absolute;
+  top: 136px;
+  left: 472px;
+  z-index: -5;
+  max-width: 160px;
 }
 
-.landing {
+.heroCircle3 {
+  position: absolute;
+  top: 584px;
+  right: 0px;
+  z-index: -5;
+  max-width: 96px;
+}
+
+.hero {
   width: 100vw;
   height: 100vh;
+  background-image: url("../static/background/heroBG.png");
+  background-blend-mode: multiply;
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  overflow: hidden;
+  z-index: 10;
```

Comment:
if z-index is defined here, what's the point of 
`section { z-index: 1 };`

remove either one hehe